main/guides/orchestration/txvsportfolio.md

```diff
@@ -0,0 +1,168 @@
+# Transactional vs. Portfolio Contracts
+
+In Agoric Orchestration, **transactional contracts** and **portfolio contracts** serve distinct use
+cases. This section explains their differences and provides examples from real contracts.
+
+| **Characteristic**        | **Transactional Contracts**   | **Portfolio Contracts**          |
+| :------------------------ | :---------------------------- | :------------------------------- |
+| **Interaction Type**      | Single-shot                   | Multi-step, ongoing              |
+| **Persistent User State** | No                            | Yes                              |
+| **User Account**          | No dedicated on-chain account | Separate sub-account or position |
+| **Example Use Case**      | Token swap, cross-chain send  | Staking, vaults                  |
+
+Table: Comparison of Transactional and Portfolio Contracts in Agoric Orchestration
+
+## Transactional Contracts
+
+**Transactional contracts** perform one-shot actions, e.g., swapping tokens or sending them from one
+chain to another. Each use is typically triggered by a single transaction, after which there is no
+long-lived state managed by the contract for that user.
+
+**Characteristics**:
+
+- **Initiation**: Usually initiated by a single transaction from the user or external source.
+- **No per-user account**: The contract does not maintain persistent user-specific state.
+- **No smart wallet requirement**: Users often do not require a dedicated on-chain account.
+- **One-shot**: The operation typically starts and finishes in a single interaction.
+
+### Example: "Send Anywhere" Contract
+
+The `send-anywhere` contract illustrates how a user can send tokens from Agoric to another chain in
+one action. Under the hood, it exposes an `invitation` that, when exercised, moves tokens once and
+does not maintain any further user-specific state.
+
+**Key Points**:
+
+- The contract exposes a single `makeSendInvitation` method that returns an Invitation.
+- Users deposit tokens into the contract just for this one transaction.
+- After the tokens are sent, the seat concludes and there is no ongoing position to manage.
+
+Below is a simplified snippet adapted from the flows (`send-anywhere.flows.js`) showing the essence
+of the transactional flow:
+
+```js
+/**
+ * Example flow for a transactional contract:
+ *
+ * @param {ZCFSeat} seat - The user's seat, including the tokens to send.
+ * @param {string} chainName - Name of the destination chain.
+ * @param {string} destAddr - Destination address on the chain.
+ */
+export const sendIt = async (
+  orch,
+  { sharedLocalAccountP, log, zoeTools },
+  seat,
+  { chainName, destAddr }
+) => {
+  // 1. Extract the single token amount from the proposal.
+  const { give } = seat.getProposal();
+  const [[_, amount]] = Object.entries(give);
```

Comment:
This "don't care" style of amount keyword is not typical. Let's show a typical usage:

```suggestion
  const { In: amount } = give;
```